superset/mcp_service/mcp_config.py

```diff
@@ -62,8 +62,9 @@
 
 # FastMCP Factory Configuration
 MCP_FACTORY_CONFIG = {
-    "name": "Superset MCP Server",
-    "instructions": None,  # Will use default from app.py
+    "name": None,  # Will derive from APP_NAME in app.py
+    "branding": None,  # Will derive from APP_NAME in app.py
+    "instructions": None,  # Will use default from app.py (parameterized with branding)
```

Comment:
We could add a test with a branded `APP_NAME` asserting that instructions reflect this change